react/__mocks__/productDetail.ts

```diff
@@ -654,9 +654,9 @@ const productDetail = {
           commertialOffer: {
             discountHighlights: [],
             teasers: [],
-            Price: 38.9,
-            ListPrice: 38.9,
-            PriceWithoutDiscount: 38.9,
+            Price: 1540.99,
+            ListPrice: 1900.99,
+            // PriceWithoutDiscount: 38.9, the current event does not send this.
```

Comment:
I double checked and confirmed we do not currently send this property:

[Link](https://github.com/vtex-apps/store/blob/46e3df3ecba20170fc3a6eebf12370969e3b5abf/react/components/ProductTitleAndPixel.tsx#L122-L168):
![CleanShot 2023-02-28 at 11 00 40](https://user-images.githubusercontent.com/381395/221876457-2dedff5c-e1df-4586-b93f-20c8634de893.png)

![CleanShot 2023-02-28 at 11 03 17](https://user-images.githubusercontent.com/381395/221876887-0751b75d-8921-4619-b6cc-55d9dbd5662d.png)


```suggestion
```